tests/cli/test_agent.py

```diff
@@ -209,26 +212,48 @@ def test_agent_start_fails(monkeypatch, cloud_api):
     assert "TEST is not a valid agent" in result.output
 
 
+@pytest.mark.parametrize("use_token", [False, True])
```

Comment:
makes total sense that you can stack these decorators, had just never seen it done, neat!

Reply:
Ah you can stack them _or_ put them in one which leads to pretty different behavior. When stacked, they are "multiplied" so all of the different combinations are easily covered.